stdlib/public/stubs/Unicode/Common/GraphemeData.h

```diff
@@ -18,89 +18,89 @@
-  0x20011070, 0x20211073, 0x2041107F, 0x60011082, 0x604110B0, 0x206110B3, 0x602110B7, 0x202110B9,
-  0x400110BD, 0x200110C2, 0x400110CD, 0x20411100, 0x20811127, 0x6001112C, 0x20E1112D, 0x60211145,
-  0x20011173, 0x20211180, 0x60011182, 0x604111B3, 0x210111B6, 0x602111BF, 0x402111C2, 0x206111C9,
-  0x600111CE, 0x200111CF, 0x6041122C, 0x2041122F, 0x60211232, 0x20011234, 0x60011235, 0x20211236,
-  0x2001123E, 0x20011241, 0x200112DF, 0x604112E0, 0x20E112E3, 0x20211300, 0x60211302, 0x2021133B,
-  0x2001133E, 0x6001133F, 0x20011340, 0x60611341, 0x60211347, 0x6041134B, 0x20011357, 0x60211362,
-  0x20C11366, 0x20811370, 0x60411435, 0x20E11438, 0x60211440, 0x20411442, 0x60011445, 0x20011446,
-  0x2001145E, 0x200114B0, 0x602114B1, 0x20A114B3, 0x600114B9, 0x200114BA, 0x602114BB, 0x200114BD,
-  0x600114BE, 0x202114BF, 0x600114C1, 0x202114C2, 0x200115AF, 0x602115B0, 0x206115B2, 0x606115B8,
-  0x202115BC, 0x600115BE, 0x202115BF, 0x202115DC, 0x60411630, 0x20E11633, 0x6021163B, 0x2001163D,
-  0x6001163E, 0x2021163F, 0x200116AB, 0x600116AC, 0x200116AD, 0x602116AE, 0x20A116B0, 0x600116B6,
-  0x200116B7, 0x2041171D, 0x20611722, 0x60011726, 0x20811727, 0x6041182C, 0x2101182F, 0x60011838,
-  0x20211839, 0x20011930, 0x60811931, 0x60211937, 0x2021193B, 0x6001193D, 0x2001193E, 0x4001193F,
-  0x60011940, 0x40011941, 0x60011942, 0x20011943, 0x604119D1, 0x206119D4, 0x202119DA, 0x606119DC,
-  0x200119E0, 0x600119E4, 0x21211A01, 0x20A11A33, 0x60011A39, 0x40011A3A, 0x20611A3B, 0x20011A47,
-  0x20A11A51, 0x60211A57, 0x20411A59, 0x40A11A84, 0x21811A8A, 0x60011A97, 0x20211A98, 0x60011C2F,
-  0x20C11C30, 0x20A11C38, 0x60011C3E, 0x20011C3F, 0x22A11C92, 0x60011CA9, 0x20C11CAA, 0x60011CB1,
-  0x20211CB2, 0x60011CB4, 0x20211CB5, 0x20A11D31, 0x20011D3A, 0x20211D3C, 0x20C11D3F, 0x40011D46,
-  0x20011D47, 0x60811D8A, 0x20211D90, 0x60211D93, 0x20011D95, 0x60011D96, 0x20011D97, 0x20211EF3,
-  0x60211EF5, 0x20211F00, 0x40011F02, 0x60011F03, 0x60211F34, 0x20811F36, 0x60211F3E, 0x20011F40,
-  0x60011F41, 0x20011F42, 0x1E13430, 0x20013440, 0x21C13447, 0x20816AF0, 0x20C16B30, 0x20016F4F,
-  0x66C16F51, 0x20616F8F, 0x20016FE4, 0x60216FF0, 0x2021BC9D, 0x61BCA0, 0x25A1CF00, 0x22C1CF30,
-  0x2001D165, 0x6001D166, 0x2041D167, 0x6001D16D, 0x2081D16E, 0xE1D173, 0x20E1D17B, 0x20C1D185,
-  0x2061D1AA, 0x2041D242, 0x26C1DA00, 0x2621DA3B, 0x2001DA75, 0x2001DA84, 0x2081DA9B, 0x21C1DAA1,
-  0x20C1E000, 0x2201E008, 0x20C1E01B, 0x2021E023, 0x2081E026, 0x2001E08F, 0x20C1E130, 0x2001E2AE,
-  0x2061E2EC, 0x2061E4EC, 0x20C1E8D0, 0x20C1E944, 0x9FE1F000, 0x8041F10D, 0x8001F12F, 0x80A1F16C,
-  0x8021F17E, 0x8001F18E, 0x8121F191, 0x8701F1AD, 0x81C1F201, 0x8001F21A, 0x8001F22F, 0x8101F232,
-  0x8061F23C, 0xB621F249, 0x2081F3FB, 0xA7A1F400, 0xA121F546, 0x8FE1F680, 0x8161F774, 0x8541F7D5,
-  0x8061F80C, 0x80E1F848, 0x80A1F85A, 0x80E1F888, 0x8A21F8AE, 0x85C1F90C, 0x8121F93C, 0xB701F947,
-  0x3EE0000, 0x2BEE0020, 0xFEE0080, 0x3DEE0100,
+static const __swift_uint32_t _swift_stdlib_graphemeBreakProperties[638] = {
```

Comment:
You don't _really_ need the dummy 0th element, but also it doesn't really matter =)

Reply:
But the 4 bytes of binary size Steve!